src/codechicken/chunkloader/ChunkLoaderManager.java

```diff
@@ -634,6 +634,11 @@ public static void addChunkLoader(IChickenChunkLoader loader) {
     }
 
     private static ChunkLoaderOrganiser getOrganiser(IChickenChunkLoader loader) {
+    	// Added bugfix here because IChickenChunkLoad.getWorld() is needed to get the WorldServer object for load(),
+    	// and this method has the last reference to the loader
+    	if(!loaded)
+    		ChunkLoaderManager.load(DimensionManager.getWorld(CommonUtils.getDimension(loader.getWorld())));
```

Comment:
Why is the loader's world not enough, why get the dimension and pull the world from the dimension manager?